content/news/2022-11-24-ESG-WP5/index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Work Package 5 of the EuroScienceGateway project
+date: "2022-11-24"
+tease: "WP5 - a showcase results from these 3 early adopters to embrace Open Science best practices."
+hide_tease: false
+tags: [esg-wp5, esg]
+subsites: [all-eu, esg]
+main_subsite: eu
+---
+
+The fifth Work Package (WP5) of the EuroScienceGateway project will demonstrate the benefits of the EuroScienceGateway services with 3 communities,
+namely (i) Biodiversity and Climate Science with the impact on human health, (ii) Materials science and (iii)
+Astrophysics. WP5 will showcase results from these 3 early adopters that will lead by example to embrace
+Open Science best practices. Each of the adopters are at a different level of maturity in their adoption of
+Galaxy for their communities with Biodiversity/Climate being the most experienced and Astrophysics the
```

Comment:
I would remove " for their communities with Biodiversity/Climate being the most experienced and Astrophysics the least experienced" this subsentence.